src/amo/components/AddonMeta.js

```diff
@@ -54,8 +54,10 @@ export class AddonMetaBase extends React.Component {
           <p className="AddonMeta-text AddonMeta-review-count">
             {reviewCount}
           </p>
-          <Rating className="AddonMeta-Rating" rating={averageRating}
-            readOnly styleName="small" />
+          <Rating className="AddonMeta-Rating"
+            rating={averageRating}
+            readOnly
+            styleName="small" />
```

Comment:
The rule relates to props on the multiline - it doesn't cover closing `>` or `/>` afaik. Will check if there's another rule that can do that.